src/monitor.rs

```diff
@@ -143,6 +143,8 @@ impl MonitorHandle {
 
     /// The monitor refresh rate used by the system.
     ///
+    /// Return `Some` if succeed, or `None` if failed, which usually happens when the monitor the window is on is removed.
+    ///
```

Comment:
Could you wrap akin other doc comments here?

Reply:
What doc comments should I put here?

Reply:
I'm not saying to put, I'm saying that you're line is too long, make it 80 chars wide like other doc comments.

Reply:
Got it.